ddtrace/propagation/http.py

```diff
@@ -679,11 +680,24 @@ def _extract(headers):
         )
 
 
+class _None_Propagation:
+    @staticmethod
+    def _extract(headers):
+        return None
+
+    # this method technically isn't needed with the current way we have HTTPPropagator.inject setup
+    # but if it changes then we might want it
+    @staticmethod
```

Comment:
should I just delete this since it's never used? Or should I just add it to the `HTTPPropagator.inject` method?

Reply:
This is a valid point, I'm on the side of thinking we should keep this method here just to keep consistency with the other Propagation style classes in this file. If we do though, I wonder if we should also add it to `HTTPPropagator.inject()` to be explicit. WDYT?

Reply:
I'm not sure, I had it there originally but then it felt sort of silly to run unneeded code every time inject runs so removed it. I don't have a strong opinion.

Reply:
Also not a strong opinion on my part. I'm interested to hear the rest of the team's stance.

Reply:
I'd prefer to keep these methods. This way we can call inject/extract on all propagators and get defined behavior.

Reply:
I think we should keep them for now in case we rewrite the code flow/style checking in the propagator.

We could also consider making a base class for these propagator styles that all need to implement to ensure consistency